index.html

```diff
@@ -30,6 +30,27 @@
 </head>
 
 <body>
+
+    <!-- NFL Banner and Team Pick -->
+    <div class="container">
+
+      <!-- Jumbotron -->
+      <div class="jumbotron" style="background-color:gray; color: black">
+        <h1 class="text-center">Football Title</h1>
+        <h3 class="text-center">Pick your Team!</h3>
+        <div class="dropdown">
+          <button class="btn btn-secondary dropdown-toggle" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+          NFL Teams
+          </button>
+          <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+          <!--<a class="dropdown-item" href="#">Action</a> example here -->  
+        </div>
+      </div>
+    </div>
+  </body>
```

Comment:
Careful not to close the body out here, since it needs to wrap the entire page contents.